app/assets/stylesheets/typography.scss

```diff
@@ -0,0 +1,49 @@
+.display {
+  font-size: 36px;
+  font-weight: 300;
+  letter-spacing: 0.4;
+  color: $text-primary;
+  font-family: $headline-fonts;
+}
+.headline {
+  font-size: 24px;
+  letter-spacing: 0.3;
+  color: $text-primary;
+  font-family: $headline-fonts;
+}
+.title {
+  font-size: 20px;
+  letter-spacing: 0.2;
+  color: $text-primary;
+  font-family: $headline-fonts;
+}
+.subheader {
+  font-size: 18px;
+  letter-spacing: 0.2;
+  color: $text-primary;
+  font-family: $headline-fonts;
+}
+.body {
```

Comment:
oooh. how about `paragraph`

it's the default 'p' styling anyway.. so we should really only need it in special cases where you want to match that style.